internal/configure/disks/dc_get.go

```diff
@@ -65,6 +73,7 @@ func (dc *DiskConfig) getStrSlice(i *comm.Item) []string {
 
 func (dc *DiskConfig) GetContainerImage() string { return dc.getString(CONFIG_GLOBAL_CONTAINER_IMAGE) }
 func (dc *DiskConfig) GetFormatPercent() int     { return dc.getInt(CONFIG_GLOBAL_FORMAT_PERCENT) }
+func (dc *DiskConfig) GetServiceMount() bool     { return dc.getBool(CONFIG_GLOBAL_SERVICE_MOUNT) }
 func (dc *DiskConfig) GetHost() []string         { return dc.getStrSlice(CONFIG_GLOBAL_HOST) }
 func (dc *DiskConfig) GetDevice() string         { return dc.getString(CONFIG_DISK_DEVICE) }
 func (dc *DiskConfig) GetMountPoint() string     { return dc.getString(CONFIG_DISK_MOUNT_POINT) }
```

Comment:
1. The variables IS named properly.
2. It has value type checking.
3. The code logic is simple.
4. self-tested.
5. Proper constants are used.

Reply:
Maybe we need some unit tests according to the 4th suggestion

Reply:
> Maybe we need some unit tests according to the 4th suggestion

Agree. Does `curveadm` have a unit test framework? We indeed need to add some UT with every change.

Reply:
> > Maybe we need some unit tests according to the 4th suggestion
> 
> Agree. Does `curveadm` have a unit test framework? We indeed need to add some UT with every change.

We use go built-in unit testing and [testify](github.com/stretchr/testify) package to testifying our codes, there are a few test code:
* [list_test.go](https://github.com/opencurve/curveadm/blob/develop/cli/command/hosts/list_test.go)
* [mount_test.go](https://github.com/opencurve/curveadm/blob/develop/internal/task/task/fs/mount_test.go)
* [kernel_test.go](https://github.com/opencurve/curveadm/blob/develop/internal/task/task/checker/kernel_test.go)
* [network_test.go](https://github.com/opencurve/curveadm/blob/develop/internal/task/task/checker/network_test.go)

We are so much lacking of unit tests and integration tests, it leads the curveadm often unstable. It would be a huge improvement if we had unit tests and integration tests to guarantee curveadm stability. but then again, i think the integration testing is not so easy to implement.

Reply:
Yeah, I noticed these test files. I thought they were just part of checker.

Reply:
That's true, integration test is more complicated, especially the automatic one. Some mocking mechanism need to be introduced for `curveadm`.